### ISSUE TITLE
Should we update the look and feel of the list view editor to the latest Umbraco List View prop editor styling

### ISSUE THREAD

**Issue by TimGeyssens** [TimGeyssens/UIOMatic#65]
Since it was already based on that but on an older version...

**Comment by TimGeyssens** [comment]
So update to this styling 
![Image of Umbraco List View](https://s12.postimg.org/o5qqw090b/Screen_Shot_2016_10_26_at_16_11_19.png)

**Comment by mattbrailsford** [comment]
Depends what versions of Umbraco we need to support. Personally I need to use it on a 7.3.3 install which was before this UI was used so we couldn't use their markup if we did. This is the problem with the back office UI changing so much :(

**Comment by mattbrailsford** [comment]
oops, didn't mean to close the issue :)

**Comment by TimGeyssens** [comment]
Ah yeah makes sense :) Not a priority works fine with the current layout

**Comment by mattbrailsford** [comment]
In this commit eb559ee7331a41e816fd0985fb6a1bb4ea6216a1 I've added Warrens umbraco version comparison util service https://gist.github.com/warrenbuckley/010aaf893551953732cc508c6b76d569 so that you can easily compare umbraco versions in angular. This could be the way we tackle the update, maintaining 2 views, one for pre 7.4 (where most of the new directives were added) and one for post 7.4.

**Comment by TimGeyssens** [comment]
Ok I see, thanks for adding! Still keeping this to 2.1 (the UI update) since I don't see that as necessary for v2

**Comment by mattbrailsford** [comment]
Agreed, just left as a note so we know how to go about it when it comes to it.

**Comment by TimGeyssens** [comment]
Perfect :)
